pkg/visitors/remove_test.go

```diff
@@ -112,3 +112,54 @@ func TestRemoveExcept(t *testing.T) {
 	}
 
 }
+
+func TestRemoveExceptBlackList(t *testing.T) {
+	// The following blacklist contains corner case:
+	// - start, mid or end of existing names but no full name
+	// - start, mid or end of existing GUID but no full GUID
+	// so it should behave as an empty blacklist, ie remove all
+	var blacklists = []string{
+		`INEXISTING_FILENAME
```

Comment:
Could you use "" with \n here? or does that not work? That would make the formatting a little nicer :)

Reply:
Right, at first I had a long list and no array... fixed